tox.ini

```diff
@@ -19,111 +19,133 @@ passenv =
 basepython = python2.7
 deps =
     -rrequirements.txt
-commonargs =
-    --with-timer
-    --with-coverage
-    --cover-tests
-    --cover-branches
-    --cover-inclusive
-    --with-flaky
-    --no-success-flaky-report
-    --exclude-dir=managed_vms/django_cloudsql
-    --exclude-dir=managed_vms/hello_world_django
-
-[testenv:reqcheck]
-commands =
-    bash -c "find . -name requirements\*.txt |\
-        xargs -P 4 -n 1 gcp-python-repo-tools check-requirements"
-whitelist_externals =
-    bash
-    find
-    xargs
-    gcp-python-repo-tools
+examples =
+    bigquery/api
+    blog/introduction_to_data_models_in_cloud_datastore
+    cloud_logging/api
+    compute/api
+    compute/autoscaler
+    datastore/api
+    managed_vms/cloudsql
+    managed_vms/datastore
+    managed_vms/disk
+    managed_vms/extending_runtime
+    managed_vms/hello_world
+    managed_vms/hello_world_compat
+    managed_vms/memcache
+    managed_vms/pubsub
+    managed_vms/static_files
+    managed_vms/storage
+    monitoring/api
+    storage/api
```

Comment:
What if instead of an 'examples' list, you had either an `exclude` or `include` list, where `run-tests.py` has corresponding flags. If `--exclude`, then look for tests everywhere but the excluded dirs. If `--include`, only look for tests in the included dirs?